JS.md

```diff
@@ -205,6 +205,26 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+### `WebAssembly.Module.sections`
+
+The `sections` function has the signature:
+
+```
+Array sections(modueObject, sectionName)
+```
+
+If `moduleObject` is not a `WebAssembly.Module` instance, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
```

Comment:
Hey, yeah, that is confusing.  In other places we just say "if `x` is not a `Y`" (no "instance" or "object"), so I'll do that here too.